cmd/cadvisor.go

```diff
@@ -132,7 +134,7 @@ func (ml *metricSetValue) Set(value string) error {
 }
 
 func init() {
-	flag.Var(&ignoreMetrics, "disable_metrics", "comma-separated list of `metrics` to be disabled. Options are 'disk', 'diskIO', 'network', 'tcp', 'udp', 'percpu', 'sched', 'process', 'hugetlb'.")
+	flag.Var(&ignoreMetrics, "disable_metrics", "comma-separated list of `metrics` to be disabled. Options are 'disk', 'diskIO', 'network', 'tcp', 'udp', 'percpu', 'sched', 'process', 'hugetlb', 'wss'.")
```

Comment:
We already have a concept which we refer to as the working set... (memory_usage_bytes - total_inactive_file).  Is there a different name we could give this?

Reply:
The metric could be named 'container_referenced_bytes' then on `disabled_metrics` there will be `referenced` option.

Reply:
That works for me, thanks!